core/server/api/v2/utils/validators/input/options.js

```diff
@@ -0,0 +1,70 @@
+const Promise = require('bluebird');
+const _ = require('lodash');
+const validation = require('../../../../../data/validation');
+const common = require('../../../../../lib/common');
+
+const GLOBAL_VALIDATORS = {
+    id: {matches: /^[a-f\d]{24}$|^1$|me/i},
+    page: {matches: /^\d+$/},
+    limit: {matches: /^\d+|all$/},
+    from: {isDate: true},
+    to: {isDate: true},
+    columns: {matches: /^[\w, ]+$/},
+    order: {matches: /^[a-z0-9_,. ]+$/i},
+    uuid: {isUUID: true},
+    slug: {isSlug: true},
+    name: {},
+    email: {isEmail: true},
+    filter: false,
+    context: false,
+    forUpdate: false,
+    transacting: false,
+    include: false,
+    formats: false
+};
+
+const validate = (config, attrs) => {
+    let errors = [];
+
+    _.each(attrs, (value, key) => {
+        if (GLOBAL_VALIDATORS[key]) {
+            if (config.queryOptionsValues && config.queryOptionsValues[key]) {
+                if (!value || value.length === 1) {
```

Comment:
In which situations would this case be triggered :thinking: ?

Reply:
If you pass `?fields=`. There was a test which tests this behaviour.
I will add a comment.

Reply:
I just removed this handling for now. The one who adds the controller and runs into it, can extend the framework